scripts/.util/tools.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
question: what is this used for Ralf?

Reply:
the scrip installs tools needed for packaging and this file contains the versions to install. that how paketo does it in their repos but in hindsight, I should just use the makefile to install to LOCALBIN